### ISSUE TITLE
WCAG-EM as an Site Owner's Accessibility Attestation — Not overlooking known issues

### ISSUE THREAD

**Issue by mgifford** [w3c/wai-wcag-em#118]
It's not quite this, but I believe that most often a WCAG-EM report will be either done by an internal team or by a team contracted by the team that owns the website. The site developers (and owners) likely will have more information about accessibility issues than will be uncovered in any given WCAG-EM review. 

WCAG-EM is a standardized way to collect information about a site which is comparable over time and between sites. However, if you can pull down the list of open accessibility tickets already in JIRA & GitHub, shouldn't it be included? The last thing you want is to get issues pushed into JIRA and then forgotten about. A WCAG-EM report should be able to highlight if there are issues that are outstanding from the previous review. 

This fits nicely into the need to consolidate different forms of data into a single report, which summarizes the current attestation as best as possible.

**Comment by hidde** [comment]
I really like this and we see organisations doing this as part of their accessibility statements, as well as ones that have a real time (ish) list of open issues in their report. However, the methodology focuses specifically on how to evaluate consistently, what to do with the results is out of scope, as it's too org-specific. I think it would makes lots of sense to talk about this in, say, meetings, presentations and blog posts about WCAG-EM, it's a good next step.
